twilio/task_router/workflow_config.py

```diff
@@ -21,6 +21,9 @@ def to_json(self):
 
     @staticmethod
     def json2obj(data):
+        # replace instances of "filter_friendly_name" with "friendly_name"
+        data = data.replace("filter_friendly_name", "friendly_name")
```

Comment:
Just going on record and saying that I have absolutely no idea what this is
and have never been involved and don't know u people. Good luck with your
software! :))

On Tue, Jan 19, 2016 at 6:11 PM, Ragil <notifications@github.com> wrote:

> In twilio/task_router/workflow_config.py
> <https://github.com/twilio/twilio-python/pull/245#discussion_r50197230>:
>
> > @@ -21,6 +21,9 @@ def to_json(self):
> >
> >      @staticmethod
> >      def json2obj(data):
> > +        # replace instances of "filter_friendly_name" with "friendly_name"
> > +        data = data.replace("filter_friendly_name", "friendly_name")
>
> this is very dangerous. If the user has filter_friendly_name in the
> expression or some other value, this will break their software.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twilio/twilio-python/pull/245/files#r50197230>.
>



-- 
Jake Witz
Member Relations & Communications
Wisconsin Union Membership Offices
(847) 999 8755